metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -135,10 +135,14 @@ final class BloopInstall(
     }
   }
 
+  // NOTE(olafur) there's a chance that we get two build change notifications in
+  // a very short period due to duplicate `didSave` and file watching
+  // notificiations. This method is synchronized to prevent asking the user
```

Comment:
```suggestion
  // notifications. This method is synchronized to prevent asking the user
```

Reply:
Fixed, good excuse to run the tests again 😄 l